app/views/moderations/actions_panel.html.erb

```diff
@@ -6,30 +6,14 @@
   </style>
 <% end %>
 
-<div class="container mod-container" style="text-align: center">
+<div class="container mod-container">
   <header class="top-header">
     <h1>Moderate Post</h1>
     <h2>Rate the quality of this post</h2>
-    <button class="circle" type="button">
+    <button class="close-actions-panel circle centered-icon" type="button">
```

Comment:
Should the Actions Panel close when this icon is clicked? I'm not sure; it wasnt happening for me.

Reply:
Yeah it should! I figured that should be part of the "interaction" part though and am holding off on it here.

Reply:
gotcha!